src/legacy/legacy_test/timer_test.go

```diff
@@ -0,0 +1,245 @@
+package legacy_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/noxworld-dev/opennox-lib/platform"
+	"github.com/noxworld-dev/opennox/v1/legacy"
+	"github.com/stretchr/testify/require"
+)
+
+type TestPlatform struct {
+	t time.Duration
+}
```

Comment:
```suggestion
type TestPlatform struct {
	T time.Duration
}
```
With field unexported, feel free to PR it against `opennox-lib`. Could be handy in other places too!

Reply:
Created https://github.com/noxworld-dev/opennox-lib/pull/6 . Originally kept it private as I could use `Ticks()` and `Sleep()` as getter/setter of the value. Anyway it's fine to have it accessible.